bin/publish-to-npm.sh

```diff
@@ -9,5 +9,5 @@ echo Current $npm_package_name $NPM_VERSION
 
 if [[ $PKG_VERSION != $NPM_VERSION ]]; then
   echo Publishing $npm_package_name $PKG_VERSION
-  npm publish --access public --registry 'https://registry.npmjs.org/'
+  npm publish --access public
```

Comment:
@TrySound i've copy-pasted all of the publish-related setups from this branch into my playground repo and it published packages successfully from the CI, just got the email from the npm:
> Successfully published andarist-changeset-action-test-a@0.0.3

I'd say that we can roll with the changes in this PR and if this ends up creating any problems, then I could investigate this further. WDYT?

Reply:
Oh, sorry - I've misinterpreted some results. Still rechecking what's wrong here.

Reply:
Ok, I've fixed the problem. It seems that `.npmrc` is the easiest way to make this work. I've always believed that it can be done solely based on the env vars but I've failed to do so right now. So I just ended up creating this manually before calling the publish script